app.py

```diff
@@ -159,7 +120,11 @@
     cluster=ecs_stack.cluster,
     props=api_props,
 )
-api_stack.add_dependency(mongo_stack)
+api_stack.add_dependency(docdb_stack)
+api_stack.service.connections.allow_to_default_port(
+    docdb_stack.cluster,
+    "Allow API container to connect to DocumentDB cluster",
```

Comment:
This is actually allowing any container in the ECS cluster to connect to the doc db.

Reply:
Nope i'm wrong, it's only allowing API container access.  sorry